spec/1.1/index.bs

```diff
@@ -711,7 +711,7 @@ Return the {{VRDisplay/displayId}} of the {{VRDisplay}} this {{Gamepad}} is asso
 
 While not directly affecting the API interface and Web IDL, WebVR implementations should maintain the user's expectations of privacy, security, and comfort on the Web by adhering to the following guidelines:
 
-* Trusted UI must be drawn by an independent rendering context who's state is isolated from that of the WebGL contexts provided as a {{VRLayerInit}} source to a {{VRDisplay}}.
+* Trusted UI must be drawn by an independent rendering context whose state is isolated from that of the WebGL contexts provided as a {{VRLayerInit}} source to a {{VRDisplay}}.
```

Comment:
I can merge this and fix the other ones. I figured I'd ask first if you wanted to do it, before I went ahead and merged it though.